grr/server/grr_response_server/gui/ui/components/flow_details/plugins/osquery_details.ng.html

```diff
@@ -12,12 +12,12 @@
     <div class="table-container">
       <table>
         <thead>
-          <th *ngFor="let column of resultTable.header.columns">{{column.name}}</th>
+          <th *ngFor="let column of resultTable.header.columns; trackBy: trackByIndex">{{column.name}}</th>
```

Comment:
We typically trackBy an object key (see https://stackoverflow.com/questions/54478273/angular-trackby-returning-the-index-in-ngfor-vs-item-id). Tracking by an index seems to be fine in this scenario, no change needed.

Reply:
I'm curious, what "key" exactly? "key" in the sense of a unique value that identifies the content of the thing and depends on the case?

Reply:
`column.name` would be a sensible key. On top of the performance benefits through DOM reuse from tracking by key/index (see [so](https://stackoverflow.com/questions/54478273/angular-trackby-returning-the-index-in-ngfor-vs-item-id)), tracking by a key works better when list items are re-ordered. We don't really re-order items, so it does not matter too much for us.